nextflow.config

```diff
@@ -22,7 +22,8 @@ params {
     gtf               = null
     transcript_fasta  = null
 
-    // salmon alevin parameters
+    // salmon alevin parameters (simpleaf)
+    simpleaf_rlen     = 91
```

Comment:
Please add this to the pipeline JSON schema

Reply:
Yep, well spotted. When all is fixed in the context of the way the workflow is, I will run the `nf-core tools` to address all changes required in the schema.

Reply:
Added